GNUmakefile

```diff
@@ -1,314 +0,0 @@
-SHELL=/bin/sh
```

Comment:
Maybe i'm weird, but i'm one of those people that doesn't usually use dub.

I also think distro packagers mostly prefer using the make file to build GtkD.

Reply:
Hm. Sounds logically. So Makefile should be. I will try to rewrite it for the changes or return existing with some fixes.